xamarin/Xamarin.Forms
# Fix UWP Toolbar icons

### Description of Change ###



Change the icons on UWP ToolbarItem to map to `AppBarButton.Content` and add an `Image` in there instead of using the `AppBarButton.Icon` directly because of a bug in UWP.

### Issues Resolved ### 


- fixes #7505

### API Changes ###

 
 None

### Platforms Affected ### 


- UWP

### Behavioral/Visual Changes ###


The toolbar icons will look a bit different and less "iconized". You can now show full colored icons.

If users are not doing so already, they can create monochromatic images to mimic the old behavior themselves.

### Before/After Screenshots ### 


As you can see in the screenshots below there is a slight difference in the look whenever the toolbar item is primary. This change also allows full color icons to be used.

You can also see a black square, that is the full color Xamarin logo. In the old way, this was not supported, in the new way it is. You can see while using the new way for a secondary item the full colored Xamarin logo is (still) a black square. To make an icon still show up when a toolbar item is secondary, we still need to assign it directly to `AppBarButton.Icon`. I have tested if the bug with the secondary monitor also affects items that are secondary, but from what I can tell it does not.

#### Original icons used in both screenshots below

![bank](https://user-images.githubusercontent.com/939291/67290956-1a71df00-f4e1-11e9-8ecb-d7fee1ccade7.png)
![calculator](https://user-images.githubusercontent.com/939291/67290958-1b0a7580-f4e1-11e9-8f66-8dc45709169b.png)

#### Old

![Old way of showing icons](https://user-images.githubusercontent.com/939291/67290513-7c7e1480-f4e0-11e9-90f6-48eb5713b67e.png)

#### New

![New way of showing icons](https://user-images.githubusercontent.com/939291/67290563-915aa800-f4e0-11e9-9342-24df7fd1291c.png)

### Testing Procedure ###


Go to Issue7505 and inspect the toolbar icons. It will take a dual monitor setup (with different resolutions) to test if the icons are not disappearing anymore. To see the old vs new appearance you would have to comment some lines in the `Platform.cs` file. Or you can compare the looks from the secondary item icons and the primary ones.

### PR Checklist ###
